modern/src/reports/ReportFilter.js

```diff
@@ -2,17 +2,27 @@ import React, { useState } from 'react';
 import {
   FormControl, InputLabel, Select, MenuItem, Button, TextField, Grid, Typography,
 } from '@material-ui/core';
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import moment from 'moment';
+import { devicesActions } from '../store';
 import t from '../common/localization';
 
-const ReportFilter = ({ children, handleSubmit, showOnly }) => {
+const ReportFilter = ({
+  children, handleSubmit, showOnly, replay,
```

Comment:
I don't think it's ideal to have feature specific parameter here. Maybe we can make the name more generic, like `fullScreen` or something like that.

Reply:
I too don't want to have this param but the layout of the Report Filter is completely different on Replay Screen.

Reply:
That's fine, but I want the variable name to be more generic because what if some other screen needs the same layout?